polychrom_hoomd/forces.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+from hoomd import md
+
+
+def kg_func(theta, kappa):
+    """Kremer-Grest bending energy penalty function"""
+
+    V = kappa * (1. + np.cos(theta))
+    T = kappa * np.sin(theta)
+
+    return V, T
+    
+    
+def poly_func(r, rmin, rmax, epsilon):
+    """Polychrom soft-core repulsion"""
+
+    term = (r* np.sqrt(6/7)) / rmax
+    
+    V = epsilon * (1 + (term**12) * (term**2 - 1) * (823543./46656.))
+    F = -epsilon * (12.0*r**13/rmax**14 + 84.0*r**11*(0.857142857142857*r**2/rmax**2 - 1)/rmax**12)
+    
+    return V, F
+
+
+def DPD_func(r, rmin, rmax, epsilon):
+    """DPD soft-core repulsion"""
+    
+    V = epsilon * rmax * (1. - r/rmax)**2
+    F = epsilon * (1. - r/rmax)
+    
+    return V, F
+    
+    
+def CSW_func(r, rmin, rmax, epsilon, rcut, n=2500, m=2500):
```

Comment:
would some other names be clearer for n&m or are these things that wouldn't really be varied so it wouldn't matter?